### PR TITLE
Docker 1.24 e2e tests

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*
Docker 1.24 e2e tests

*Testing (if applicable):*
- make e2e

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vivek-koppuru — APPROVED** [review]

File: test/e2e/flux_test.go [hunk 1]
```diff
@@ -45,20 +45,22 @@ func TestDockerKubernetes120FluxLegacy(t *testing.T) {
 	runFluxFlow(test)
 }
 
-func TestDockerKubernetes123GithubFlux(t *testing.T) {
+func TestDockerKubernetes124GithubFlux(t *testing.T) {
```
Comment (jiayiwang7) [inline test/e2e/flux_test.go:48]: we don't need the 1.23 tests?

Comment (mitalipaygude) [reply test/e2e/flux_test.go:48]: Not all of them.

**Pull request merged**